bcda-site-static/index.md

```diff
@@ -19,53 +21,56 @@ ctas:
+  
+  As of January 2019, the Beneficiary Claims Data API is a work in progress.
 
-    * We’re currently providing synthetic data that resembles the kinds of data Shared Savings Program ACOs will receive by connecting with this endpoint, so that they can try out different ways of interacting with the API before receiving live data.
+   * We’re currently providing synthetic data that resembles the kinds of data Shared Savings Program ACOs will receive by connecting with this endpoint, so that they can try out different ways of interacting with the API before receiving live data.
 
-    * We’re providing this test experience and documentation early so we can learn from Shared Savings Program ACOs and their vendor partners who need to access this information, about what works best for them. Through conversations and test drives, we strive to learn what ACOs need from bulk beneficiary claims data, and create a process that meets their needs.
+   * We’re providing this test experience and documentation early so we can learn from Shared Savings Program ACOs and their vendor partners who need to access this information, about what works best for them. Through conversations and test drives, we strive to learn what ACOs need from bulk beneficiary claims data, and create a process that meets their needs.
 
-    * Developers, analysts, and administrators at Shared Savings Program ACOs are invited to participate in this pilot process by joining our Google Group.
+   * Developers, analysts, and administrators at Shared Savings Program ACOs are invited to participate in this pilot process by joining our Google Group.
 
-  * What has been learned from ACOs in the pilot?
+## What has been learned from ACOs in the pilot?
+{:#what-we-ve-learned}
 
-    Developers, analysts, and administrators at ACOs have been instrumental in shaping CMS’ approach to this API. With their feedback, the team is implementing the following elements:
+   Developers, analysts, and administrators at ACOs have been instrumental in shaping CMS’ approach to this API. With their feedback, the team is implementing the following elements:
 
-    * Providing clear, human-readable narrative documentation to aid all users’ use of the API and the data that is shared
-    * Using resilient NDJSON ([New Line Delimited JSON](http://ndjson.org)) rather than fixed-width files in response to requests for delimited information
-    * Providing bulk beneficiary claims data through an automated retrieval process that requires minimal hands-on intervention to receive
-    * Formatting data in accordance with robust Fast Health Interoperability Resource ([FHIR](https://www.hl7.org/fhir/overview.html)) specifications
+   * Providing clear, human-readable narrative documentation to aid all users’ use of the API and the data that is shared
```

Comment:
Nit-picking: the bullet points in this section do not end with any punctuation.

Reply:
this is per the source doc from Elissa.  I am happy to change, but need approval from her.